In [1]:
import numpy as np
import pandas as pd
from scipy import stats as st
import seaborn as sns 
import statistics as stats

In [2]:
df = pd.read_csv('Q7.csv')
df.head()

Unnamed: 0  Points  Score  Weigh
0          Mazda RX4    3.90  2.620  16.46
1      Mazda RX4 Wag    3.90  2.875  17.02
2         Datsun 710    3.85  2.320  18.61
3     Hornet 4 Drive    3.08  3.215  19.44
4  Hornet Sportabout    3.15  3.440  17.02

In [3]:
pd.DataFrame(df['Points']).mean()

Points    3.596563
dtype: float64

In [4]:
pd.DataFrame(df['Points']).median()

Points    3.695
dtype: float64

In [5]:
import warnings as warnings
warnings.filterwarnings('ignore')

df.std()

Points    0.534679
Score     0.978457
Weigh     1.786943
dtype: float64

In [7]:
df.var()

Points    0.285881
Score     0.957379
Weigh     3.193166
dtype: float64

Points mean = 3.59656

Points median = 3.695

Points std = 0.53467

Points var = 0.285881

In [8]:
pd.DataFrame(df['Score']).mean()

Score    3.21725
dtype: float64

In [9]:
pd.DataFrame(df['Score']).median()

Score    3.325
dtype: float64

In [10]:
df.std()

Points    0.534679
Score     0.978457
Weigh     1.786943
dtype: float64

In [11]:
df.var()

Points    0.285881
Score     0.957379
Weigh     3.193166
dtype: float64

Score mean = 3.21725

Score median = 3.325

Score std = 0.978457

Score var = 0.957379

In [12]:
pd.DataFrame(df['Weigh']).mean()

Weigh    17.84875
dtype: float64

In [13]:
pd.DataFrame(df['Weigh']).median()

Weigh    17.71
dtype: float64

In [14]:
df.std()

Points    0.534679
Score     0.978457
Weigh     1.786943
dtype: float64

In [15]:
df.var()

Points    0.285881
Score     0.957379
Weigh     3.193166
dtype: float64

Weigh mean = 17.84875

Weigh median = 17.71

Weigh std = 1.786943

weifg var = 3.193166
 
 